openapi/clusters_mgmt/v1/openapi.json

```diff
@@ -3822,6 +3858,112 @@
         }
       }
     },
+    "/api/clusters_mgmt/v1/provision_shards": {
+      "get": {
+        "parameters": [
+          {
+            "name": "page",
+            "description": "Index of the requested page, where one corresponds to the first page.\n",
+            "in": "query",
+            "schema": {
+              "type": "integer",
+              "format": "int32"
+            }
+          },
+          {
+            "name": "size",
+            "description": "Maximum number of items that will be contained in the returned page.\n",
+            "in": "query",
+            "schema": {
+              "type": "integer",
+              "format": "int32"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success.\n",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "items": {
+                      "description": "Retrieved list provision shards.\n",
```

Comment:
np: I know it's generated but I just noticed it's missing `of`

Reply:
Fixed in a followup MR.